import-service/src/upload.js

```diff
@@ -20,16 +22,28 @@ const importFileParser = async (event) => {
       Key: key,
     }).createReadStream();
 
-    return new Promise((resolve, reject) => {
+    const processedData = []
+
+    await new Promise((resolve, reject) => {
       s3Stream
         .pipe(csv())
-        .on('data', (data) => {
-          console.log("csv data: ");
-          console.log(data);
+        .on('data', async (data) => {
+          if (Object.keys(data).length === 3 && data['_1'] !== 'title') {
```

Comment:
Not sure why it is needed here. After you parse a csv file you already have normal object
https://www.npmjs.com/package/csv-parser
And you can push date to sqs queue to omit additional loop in the future with magic number 6

Reply:
i was getting additional csv meta data which needs to be filtered

Reply:
it is strange, may be you have wrong csv file, or wrong library
https://www.npmjs.com/package/csv-parser#usage

Usage
To use the module, create a readable stream to a desired CSV file, instantiate csv, and pipe the stream to csv.

Suppose you have a CSV file data.csv which contains the data:

NAME,AGE
Daffy Duck,24
Bugs Bunny,22
It could then be parsed, and results shown like so:

const csv = require('csv-parser')
const fs = require('fs')
const results = [];

fs.createReadStream('data.csv')
  .pipe(csv())
  .on('data', (data) => results.push(data))
  .on('end', () => {
    console.log(results);
    // [
    //   { NAME: 'Daffy Duck', AGE: '24' },
    //   { NAME: 'Bugs Bunny', AGE: '22' }
    // ]
  });